Formula/tinyproxy.rb

```diff
@@ -43,7 +43,7 @@ def install
 
     # Fix broken XML lint
     # See: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=154624
-    inreplace %w[docs/man5/Makefile docs/man8/Makefile"], "-f manpage",
+    inreplace %w[docs/man5/Makefile docs/man8/Makefile], "-f manpage",
                                                           "-f manpage \\\n  -L"
```

Comment:
This would be 92 chars on one line, I would shoot for 80 in general unless there are URLs and stuff.

I think it's fine to just align the `"-f`'s as it was like before.